recipes/kim_convergence/LICENSE

```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
This file is already included in the 0.0.2 tarball from PyPI, so you should not include that file manually in the recipe.

Reply:
@dopplershift thanks, I removed the file.